distribution.pm

```diff
@@ -130,15 +131,16 @@ sub script_run {
     my ($self, $cmd) = splice(@_, 0, 2);
     my %args = testapi::compat_args(
         {
-            timeout => $bmwqemu::default_timeout,
-            output  => '',
-            quiet   => undef
+            timeout      => $bmwqemu::default_timeout,
+            output       => '',
+            quiet        => undef,
+            max_interval => undef
```

Comment:
this one would mangle the input aswell

Reply:
but we need those default values, don't we?

Reply:
@foursixnine, why? do you have an example that rejects the hypothesis supported by my pasted script?